sql/moz-fx-data-shared-prod/telemetry_derived/main_summary_v4/part1.sql

```diff
@@ -355,7 +355,10 @@ SELECT
   ) AS events,
   -- bug 1339655
   SAFE_CAST(
-    JSON_EXTRACT_SCALAR(payload.histograms.ssl_handshake_result, '$.values.0') AS INT64
+    mozfun.map.get_key(
+      mozfun.hist.extract(payload.histograms.ssl_handshake_result).values,
```

Comment:
I'm a little nervous about adding several new calls to `mozfun.hist.extract` and `mozfun.map.get_key` here, which are going to increase query complexity. If this query already had a preliminary CTE, I would suggest doing a single `mozfun.hist.extract` per histogram there, and reusing the value here. This is probably fine as long as dry runs can succeed.